omnisend/includes/Internal/class-emailevents.php

```
@@ -0,0 +1,230 @@
+	}
+
+	/**
+	 * Event when user email change request is initiated
+	 *
+	 * @param int     $user_id,
+	 * @param WP_User $old_user_data
+	 * @param array   $user_data
+	 *
+	 * @return void
+	 */
+	public function user_email_change_event( int $user_id, WP_User $old_user_data, array $user_data ): void {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$old_user_data'. (Unused Code Rules)

(UnusedFormalParameter)